rskj-core/src/main/java/co/rsk/peg/whitelist/WhitelistSupport.java

```diff
@@ -0,0 +1,24 @@
+package co.rsk.peg.whitelist;
+
+import java.math.BigInteger;
+import org.ethereum.core.Transaction;
+
+/**
+ * Interface for the whitelist support. This interface is used to add, remove and get whitelist entries.
+ */
+public interface WhitelistSupport {
+
+    Integer getLockWhitelistSize();
```

Comment:
Can the methods that return a boxed primitive type be changed to just a primitive instead? This way we avoid doing null checks.

Reply:
Agree